internal/service/alertconfiguration/resource_test.go

```diff
@@ -1039,6 +1078,35 @@ func configWithEmptyOptionalBlocks(projectID string) string {
 	`, projectID)
 }
 
+func configWithSeverityOverride(projectID string, severity *string) string {
+	severityOverride := ""
+	if severity != nil {
+		severityOverride = fmt.Sprintf("severity_override = %[1]q", *severity)
+	}
+
+	x := fmt.Sprintf(`
+		resource "mongodbatlas_alert_configuration" "test" {
+			project_id        = %[1]q
+			enabled           = true
+			event_type        = "NO_PRIMARY"
+			%[2]s
+
+			notification {
+				type_name     = "EMAIL"
+				interval_min  = 60
+				email_address = "test@mongodbtest.com"
+			}
+		}
+
+		data "mongodbatlas_alert_configuration" "test" {
+			project_id             = mongodbatlas_alert_configuration.test.project_id
+			alert_configuration_id = mongodbatlas_alert_configuration.test.id
+		}
+		`, projectID, severityOverride)
+
+	return x
```

Comment:
```suggestion
	return fmt.Sprintf(`
		resource "mongodbatlas_alert_configuration" "test" {
			project_id        = %[1]q
			enabled           = true
			event_type        = "NO_PRIMARY"
			%[2]s

			notification {
				type_name     = "EMAIL"
				interval_min  = 60
				email_address = "test@mongodbtest.com"
			}
		}

		data "mongodbatlas_alert_configuration" "test" {
			project_id             = mongodbatlas_alert_configuration.test.project_id
			alert_configuration_id = mongodbatlas_alert_configuration.test.id
		}
		`, projectID, severityOverride)
```

can just return instead of "inventing" a variable name

Reply:
leftover from debugging :), thanks